docker/docker-bootstrap.sh

```diff
@@ -18,9 +18,12 @@
 
 set -eo pipefail
 
+# Ensure the temporary directory exists
+mkdir -p /tmp/superset_temp
+
 # Make python interactive
 if [ "$DEV_MODE" == "true" ]; then
-    if command -v uv > /dev/null 2>&1; then
+    if [ "$(whoami)" = "root" ] && command -v uv > /dev/null 2>&1; then
```

Comment:
only doing this when `root` as `superset` doesn't have the rights to touch the python packaging.